lib/course_planner_web/controllers/user_controller.ex

```diff
@@ -75,4 +77,19 @@ defmodule CoursePlannerWeb.UserController do
     |> put_flash(:info, "Users notified successfully.")
     |> redirect(to: user_path(conn, :index))
   end
+
+  def resend_email(conn, %{"id" => id}) do
+    token = ControllerHelpers.random_string 48
+    url = Helpers.password_url(conn, :edit, token)
+    case Users.get(id) do
+      {:ok, user} -> ControllerHelpers.send_user_email(:welcome, user, url)
```

Comment:
can you put `ControllerHelpers.send_user_email(:welcome, user, url)` in the next line? this is a bit confusing